src/main/resources/templates/views/home.html

```diff
@@ -128,6 +128,9 @@ <h5>3rd slide Headline</h5>
                         <li><a class="dropdown-item" th:href="@{/Wahlpflicht}">Wahlpflicht</a></li>
                     </ul>
                 </li>
```

Comment:
Statt me-5 bitte me-auto verwenden, damit es in mobile auch gut aussieht.
Auch bitte einmal prüfen, ob der Button nur angezeigt wird, wenn auch ein Filter aktiv ist. Vielleicht mittels th:if